apertium-uzb.uzb.lexc

```diff
@@ -3392,12 +3397,12 @@ bariton:bariton N1 ; !
 barkamol:barkamol A2 ; ! 
 barkastalik:barkastalik N1 ;  ! 
 barkash:barkash N1 ; ! 
-barqaror:barqaror A1 ; ! 
-barqarorlik:barqarorlik N1 ;  ! 
+barqaror:barqaror A1 ; ! "stabil" 
```

Comment:
"stable"

Reply:
Hah, looks like even-though I was born after the Uzbekistan's independence, deep down inside, I have some Russian instincts :D . Fixed it.